components/eam/src/physics/cam/tropopause.F90

```diff
@@ -1206,6 +1206,8 @@ subroutine tropopause_e90_3d(pstate, tropLevB, tropLevU, tropFlag, tropFlagInt,
          do k = 2, pver
             ! Search starts at the top and stops the first time
             ! E90 greater than thrd.
+            tropLevU(i) = tropLevB(i) !added by kzm to avoid no value found at first step
+            tropLevU(i) = max(2,tropLevU(i)) !added by kzm to avoid no value found at first step
```

Comment:
"at first step" is not clear. Do you mean the first round in this do-loop?

Reply:
@tangq in this loop, the tropLevU would have no value at all, if situation can't be satisfied. (pstate%q(i,k,e90_ndx) > thrd_mmr). I changed the comment.